products/sql/ansible.yaml

```diff
@@ -44,9 +44,39 @@ overrides: !ruby/object:Provider::ResourceOverrides
           result = response.json()
       except getattr(json.decoder, 'JSONDecodeError', ValueError) as inst:
           module.fail_json(msg="Invalid JSON response with error: %s" % inst)
-  # Objects not implmented yet.
+
+      if navigate_hash(result, ['error', 'errors']):
+          module.fail_json(msg=navigate_hash(result, ['error', 'errors']))
+      if result['kind'] != kind:
+          module.fail_json(msg="Incorrect result: {kind}".format(**result))
+
+      return result
   Database: !ruby/object:Provider::Ansible::ResourceOverride
-    exclude: true
+    return_if_object: |
+      # If not found, return nothing.
+      if response.status_code == 404:
+          return None
+
+      # If no content, return nothing.
+      if response.status_code == 204:
+          return None
+
+      # SQL only: return on 403 if not exist
+      if response.status_code == 403:
+          return None
+
+      try:
+          result = response.json()
+      except getattr(json.decoder, 'JSONDecodeError', ValueError) as inst:
+          module.fail_json(msg="Invalid JSON response with error: %s" % inst)
+
+      if navigate_hash(result, ['error', 'errors']):
+          module.fail_json(msg=navigate_hash(result, ['error', 'errors']))
+      if result['kind'] not in [kind, 'sql#instance']:
+          module.fail_json(msg="Incorrect result: {kind}".format(**result))
+
+      return result
+  # Objects not implmented yet.
```

Comment:
These are referring to GCP resources that I haven't officially built support for yet.

(We call them objects at time because the `object` variable is used to describe a resource)